go-server-server/go/default.go

```diff
@@ -280,8 +280,13 @@ func ConfigInterfaceVlanPost(w http.ResponseWriter, r *http.Request) {
         return
     }
     if vlan_kv != nil {
-        WriteRequestErrorWithSubCode(w, http.StatusConflict, RESRC_EXISTS,
-              "Object already exists: " + vlan_name, []string{}, "")
+        vlan_if_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VLAN_INTF_TB, vlan_name))
+        if vlan_if_kv != nil {
+                vnet_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VNET_TB, vlan_if_kv["vnet_name"]))
+                attr.Vnet_id = vnet_kv["guid"]
+                WriteRequestErrorWithSubCode(w, http.StatusConflict, RESRC_EXISTS,
+                    "Object already exists: {\"vlan_name\":\"" + vlan_name + "\", \"vnet_id\":\"" + vnet_kv["guid"] +"\"}", []string{}, "")
+        }
```

Comment:
please have the old error message in the 'else' part. Lets not return empty if vlan_if_kv is nil